code/modules/religion/sects/shadow_sect.dm

```diff
@@ -319,93 +324,91 @@
+	desc = "Activates the obelisk heal and (if you meet the requirements) summons Faithsworn entities from 30% of the obelisks in existence. THIS IS ONE USE ONLY."
 	ritual_length = 60 SECONDS
 	ritual_invocations = list(
 		"Join us in this darkness ...",
 		"... Protect us from the light ...",
 		"... Destroy those who defy us ...")
 	invoke_msg = "... Let the darkness come and fight!!"
-	favor_cost = 50000
+	favor_cost = 25000
 
 /datum/religion_rites/final_darkness/invoke_effect(mob/living/user, atom/religious_tool)
 	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
 	if(sect.faithful_used)
 		to_chat(user,"<span class='warning'>This rite has already been used, your favor has been refuned.</span>")
-		GLOB.religious_sect?.adjust_favor(50000, user)
+		GLOB.religious_sect?.adjust_favor(favor_cost, user)
 		return ..()
+	sect.murder = user.mind.is_murderbone() || sect.murder
 	sect.faithful_used = TRUE
 	var/altar_turf = get_turf(religious_tool)
 	var/obj/structure/destructible/religion/shadow_obelisk/lisk = new(altar_turf)
 	sect.obelisks += lisk
 	lisk.AddComponent(/datum/component/dark_favor, user)
 	lisk.set_light(sect.light_reach, sect.light_power, DARKNESS_INVERSE_COLOR)
 	playsound(altar_turf, 'sound/magic/fireball.ogg', 50, TRUE)
-	priority_announce("May our lord, [GLOB.deity], have mercy on your soul as darkness reigns apon you all.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
-	addtimer(CALLBACK(lisk, TYPE_PROC_REF(/obj/structure/destructible/religion/shadow_obelisk, final_darkness_activate)), 100)
+	if(sect.murder)
+		priority_announce("May our lord, [GLOB.deity], have mercy on your soul as darkness reigns apon you all.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
+	else
+		priority_announce("May our lord, [GLOB.deity], bless all the shadows as darkness heals those who worship the night.", "Faith Alert", SSstation.announcer.get_rand_alert_sound())
+	addtimer(CALLBACK(lisk, TYPE_PROC_REF(/obj/structure/destructible/religion/shadow_obelisk, final_darkness_activate)), 250)
 
 /obj/structure/destructible/religion/shadow_obelisk/proc/final_darkness_activate()
 	var/datum/religion_sect/shadow_sect/sect = GLOB.religious_sect
+	var/list/candidates = poll_ghost_candidates("Do you wish to be summoned as a Shadow Faithful?", ROLE_HOLY_SUMMONED, null, 10 SECONDS, POLL_IGNORE_HOLYUNDEAD)
 	for(var/obj/structure/destructible/religion/shadow_obelisk/obs in sect.obelisks)
 		START_PROCESSING(SSobj, obs)
+		if(!sect.murder)
+			playsound(obs, 'sound/magic/fireball.ogg', 50, TRUE)
+			return
 		var/obelisk_turf = get_turf(obs)
-		for(var/i in 1 to 3)
-			var/mob/living/simple_animal/hostile/faithless/faithful/faithful = new(obelisk_turf)
-			faithful.AddComponent(/datum/component/dark_favor, faithful)
-			faithful.set_light(2, -2, DARKNESS_INVERSE_COLOR)
+		if(prob(30))
+			for(var/i in sect.min_summon to sect.max_summon)
```

Comment:
Since you're here, let's fix it too. It shouldn't be `/obj` proc. It should be `/datum` since it's a ritual.

Reply:
I did originally try this, but as the rite end, I.E Invoke effect finishes, it seems to delete itself or something, cancelling any callbacks/timers and so on that would be in progress, so I adjusted it to be on the obelisk it spawns to counteract this